pkg/protocol/engine/utxoledger/output.go

```diff
@@ -173,8 +173,8 @@ func (o *Output) KVStorableKey() (key []byte) {
 }
 
 func (o *Output) KVStorableValue() (value []byte) {
-	ms := marshalutil.New(48)
-	ms.WriteBytes(o.blockID[:])              // 32 bytes
+	ms := marshalutil.New(56)
```

Comment:
I wouldn't specify the length here at all. That's not the correct length if `o.encodedOutput` is > 0 and thus the slice needs to be reallocated/expanded anyway. Then better use the default 1024 of the marshalutil